src/app/left-menu/main/left-menu.component.ts

```diff
@@ -63,7 +63,7 @@ export class LeftMenuComponent implements OnDestroy, OnInit {
     }
 
     changeWidget(widget: string) {
-        const item = this.menuData && this.menuData.find(value => value.groupKey === widget);
+        const item = this.menuData && this.menuData.find(value => value.groupKey === (widget === 'treelist' ? 'datagrid': widget));
```

Comment:
Widget's names here will stop the refactoring of this component. As you see it need urgent refactoring.